tests/search-resolver.tests.ts

```diff
@@ -104,7 +104,7 @@ describe('Pagination browser test', () => {
       apiKey: '',
       clientAgents: [`Meilisearch instant-meilisearch (v${PACKAGE_VERSION})`],
     })
-    expect(mockedSearch).toHaveBeenCalledTimes(2)
+    expect(mockedSearch).toHaveBeenCalledTimes(3)
```

Comment:
Why do we have +1 calls now?

Reply:
Because the first call always sets up the default facet distribution by making a call with zero filters (except for the facet field) to ensure we have the facet distribution of an empty search not impacted by any filter.

Reply:
See https://github.com/meilisearch/instant-meilisearch/pull/830#discussion_r959370174